concent_api/utils/message_handler.py

```diff
@@ -0,0 +1,72 @@
+from functools                      import wraps
+
+from django.db                      import transaction
+from django.http                    import JsonResponse
+from django.http                    import HttpResponse
+from django.http                    import HttpResponseNotAllowed
+from django.conf                    import settings
+
+from golem_messages                 import dump
+from golem_messages.message         import Message
+
+from core.exceptions                import Http400
+
+from utils                          import logging
+
+
+def message_handler(view):
+    @wraps(view)
```

Comment:
Please add a comment what this decorator does, and as with previous one write some tests if possible (and reuse existing ones for `api_view`).